src/library.js

```diff
@@ -1225,9 +1225,9 @@ mergeInto(LibraryManager.library, {
 
     return 0;
   },
-  strptime_l__sig: 'pppp',
+  strptime_l__sig: 'ppppp',
   strptime_l__deps: ['strptime'],
-  strptime_l: function(buf, format, tm) {
+  strptime_l: function(buf, format, tm, locale) {
```

Comment:
I can't seem to find the `_l` variant anywhere in `system/*`. What am I missing?

Reply:
https://man.freebsd.org/cgi/man.cgi?query=strptime_l&apropos=0&sektion=3&manpath=FreeBSD+11-current&format=html